examples/market_maker/on_chain_market_maker.vy

```diff
@@ -8,7 +8,7 @@ totalTokenQty: public(uint256)
 # Constant set in `initiate` that's used to calculate
 # the amount of ether/tokens that are exchanged
 invariant: public(uint256)
-token_address: IERC20
+token: IERC20
```

Comment:
Changed the storage variable name to better reflect it is an interface and not an address